### DISCUSSION TITLE
Cant access telegram bot token.

### DISCUSSION THREAD

**Discussion by jamregistr** [aahnik/tgcf#338] [Q&A]
How can i access my configuration from telegram bot token? My bot is working perfect, but i cannot control him through telegram bot token. I added my account id at admins: [12345678] and at .env file i add variable with telegram_bot_token=1324125125:dfdsgafgasgasvasf. My configuration looks absolutely the same as the guide, but my bot chat is absolutely empty and nothing happens after /start command.

**Comment by eliafino** [comment]
Have you start tgcf and configured your api id, hash and bot token?
Linux? Have you exported env variable?

**Reply by jamregistr** [reply]
yeap, i did everything and script is working fine. I can sforward messages between chats, but i cant check script status or change yml configuration through the telegram bot token.
